src/jbuild.ml

```diff
@@ -375,6 +375,8 @@ module Lib_dep = struct
             loop required (String_set.add s forbidden) l
           else
             loop (String_set.add s required) forbidden l
+        | String _ :: _ ->
+           of_sexp_error sexp "do not quote library names"
```

Comment:
Why is it an issue to quote library names? This feels a bit arbitrary if we allow the name to be quoted in `(library ((name "foo") ...))`

Reply:
OK.